spec/factories/sites.rb

```diff
@@ -24,7 +24,7 @@
     address2 { Faker::Address.community }
     address3 { Faker::Address.city }
     address4 { Faker::Address.state }
-    postcode { Faker::Address.postcode }
+    postcode { 'SW1A 2AA' }
```

Comment:
Maybe set the Faker locale for this? Seems easy to do: https://github.com/stympy/faker/blob/master/lib/locales/README.md

Reply:
DOne